src/MarkdownClientCapabilities.php

```diff
@@ -7,43 +7,43 @@
 use RuntimeException;
 
 /**
- * Represents reasons why a text document is saved.
+ * Client capabilities specific to the used markdown parser.
  */
-class TextDocumentWillSaveEvent
```

Comment:
This protocol class no longer seems to have been transpiled - but it's not used by Phpactor